### PR TITLE
Block new AD (turnaa.png) for dizibox.pro

### PR DESCRIPTION
**Pull request by kelebek333**
Block new AD (turnaa.png) for dizibox.pro

[//]: # (***You can delete or ignore strings starting with "[//]:" They will not be visible either way.)

***Description***:
* **Current behaviour**: 

[//]: # (Substitute this line with a description of the problem)

[//]: # (Replace %screenshot_url% below with a link to the screenshot of the problem. Also, you can paste image from clipboard instead. It will be automatically loaded.)
<details><summary>Screenshot:</summary>

dizibox-ad-turna

</details><br/>

* **Expected behaviour**: 

[//]: # (Substitute this line with a description of what should happen normally. If needed, provide a screenshot below, same as above)

<details><summary>Screenshot:</summary>

dizibox-non-ad-turna

</details><br/>

***Steps to reproduce the problem***:

[//]: # (Substitute this line with a step-by-step instruction how to reproduce the issue)

***System configuration***

**Filters:**

[//]: # (Substitute this line with the list of your active filters, separated by commas)

[//]: # (Please enter the correct values for your case to the table below)

Information                            | Value
---                                    | ---
Operating system:                      | Linux
Operating system version (Android/iOS) | ?
Browser:                               | Waterfox
AdGuard version:                       | ?
Filters enabled:                       | ?
AdGuard mode (Android only):           | VPN / Proxy (manual/auto)
Filtering quality (Android only):      | High-quality / High-speed / Simplified
HTTPS filtering (Android only):        | On (Default/Blacklist mode) / Off
Simplified filters (iOS only)          | On / Off
AdGuard DNS:                           | None / Default / Family Protection
Stealth mode options (Windows only)    | ?
Helpdesk ID (if exists):               | ?

[//]: # (This template is meant for missed ad/false positive reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -428,6 +428,7 @@ koinmedya.com###custom_html-52
 ||i.ibb.co/2Pt8tqJ/turna.png
 ||i.ibb.co/0nCpbCv/otelz.jpg
 ||i.ibb.co/bXc58hB/turna.png
+||i.ibb.co/tJ9cJVx/turnaa.png
```
Comment (zloyden) [inline TurkishFilter/sections/specific.txt:431]: Can you show this picture via opened devtools? The video haven't worked on my end.
You can also use this rule and remove similar ones.
```
||i.ibb.co/*/turna$image,domain=dizibox.pro
```

Comment (kelebek333) [reply TurkishFilter/sections/specific.txt:431]: Did you try that? https://www.dizibox.pro/maedeudok-1-sezon-4-bolum-izle/
or that. https://yabancidizi.pw/dizi/maedeudok/sezon-1/bolum-4
This AD is using on more than one website.

**Review by Alex-302 — APPROVED** [review]

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -425,9 +425,8 @@ koinmedya.com##a[href^="https://beincrypto.com/"]
 pembetv18.com,kriptoparafiyatlari.com,koinmedya.com##a[href^="https://www.binance.com/"]
 koinmedya.com###custom_html-52
 ||dizibox.*/r/
-||i.ibb.co/2Pt8tqJ/turna.png
 ||i.ibb.co/0nCpbCv/otelz.jpg
-||i.ibb.co/bXc58hB/turna.png
+||i.ibb.co/*/turna*$image
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:429]: second `*` is redundant

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/specific.txt (hunk 2 above)
Comment (kelebek333) [reply TurkishFilter/sections/specific.txt:429]: There are two different file name from each other. Someone is turnaa.png and the other turna.png. They both belong to the same advertisement, but they have different file names. With the odd line is both unblocked. Thus, two ads are blocked by single line.

Comment (kelebek333) [reply TurkishFilter/sections/specific.txt:429]: https://i.ibb.co/bXc58hB/turna.png
https://i.ibb.co/tJ9cJVx/turnaa.png

Comment (Alex-302) [reply TurkishFilter/sections/specific.txt:429]: `||i.ibb.co/*/turna` covers both cases.
Or maybe
```
||i.ibb.co/*/turna.png
||i.ibb.co/*/turnaa.png
```

Comment (kelebek333) [reply TurkishFilter/sections/specific.txt]: Thanks for the explanation. I updated PR.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**